site/src/components/Resources/Resources.tsx

```diff
@@ -102,9 +106,11 @@ export const Resources: FC<ResourcesProps> = ({
                         {agent.name}
                         <div className={styles.agentInfo}>
                           <span className={styles.operatingSystem}>{agent.operating_system}</span>
-                          <span style={{ color: agentStatus.color }} className={styles.status}>
-                            {agentStatus.status}
-                          </span>
+                          {WorkspaceStateEnum[workspaceStatus] !== WorkspaceStateEnum["stopped"] && (
+                            <span style={{ color: agentStatus.color }} className={styles.status}>
```

Comment:
@jsjoeio I just did that to get you to admit you like enums ;)

Reply:
Dang it...I didn't see that one coming 😂

The Battle of Enums
Kira: 1
Joe: 0